plugins/components/index.js

```diff
@@ -1,2 +1 @@
 export { default as PluginArea } from './plugin-area';
-export { default as PluginContextProvider } from './plugin-context-provider';
```

Comment:
? :)

Reply:
I forgot to leave a note for reviewers. It isn't used outside of plugins or documented anywhere so removing it for clarity.